Source/WebCore/platform/graphics/SourceBufferPrivate.cpp

```diff
@@ -422,33 +426,115 @@ size_t SourceBufferPrivate::platformEvictionThreshold() const
     return 0;
 }
 
+Ref<GenericPromise> SourceBufferPrivate::setMaximumBufferSize(size_t size)
+{
+    if (size != m_evictionData.maximumBufferSize) {
+        m_evictionData.maximumBufferSize = size;
+        computeEvictionData(true);
```

Comment:
Would read better with a `bool forceNotification = true;` or using an enum class instead of a bool